Docs/debugging.md

```diff
@@ -0,0 +1,77 @@
+# Debugging
+
+The content of this guide will help you to configure your PHPStorm in order to debug the PIM.
```

Comment:
`to configure PHPStorm in order to debug a Symfony application.`
The doc of the GitHub repository is decoupled from the PIM as much as possible.